### ISSUE TITLE
As a group admin, I want to remove a user from a group using the member list

### ISSUE THREAD

**Issue by jrainville** [status-im/status-desktop#16130]
### Description

Currently, the only way to remove someone, as an admin, from a group is to click the kebab menu at the top and select the "Add/remove from group" option.

Discord makes it simpler by being able to right click on a user in the member list and remove from there.

I suggest putting the option above "Mark as untrusted". I would call it "Remove from group"
![image](https://github.com/user-attachments/assets/da534231-e618-461b-9da8-cda4eb22b9ae)

It could even be put in a different section (new separator)
cc @benjthayer for your ideas.

### Acceptance Criteria

- As an admin, right-clicking a member of a group chat in the member list shows the option to remove them from the group
- Clicking that new option removes that user
- Usual member removal stuff happens:
  - The user can no longer send messages to the chat or see new messages
  - People see updated list of members

**Comment by benjthayer** [comment]
> ### Description
> Currently, the only way to remove someone, as an admin, from a group is to click the kebab menu at the top and select the "Add/remove from group" option.
> 
> Discord makes it simpler by being able to right click on a user in the member list and remove from there.
> 
> I suggest putting the option above "Mark as untrusted". I would call it "Remove from group" ![image](https://private-user-images.githubusercontent.com/11926403/358292445-da534231-e618-461b-9da8-cda4eb22b9ae.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjM3Mzg5ODQsIm5iZiI6MTcyMzczODY4NCwicGF0aCI6Ii8xMTkyNjQwMy8zNTgyOTI0NDUtZGE1MzQyMzEtZTYxOC00NjFiLTlkYTgtY2RhNGViMjJiOWFlLnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA4MTUlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwODE1VDE2MTgwNFomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTI1MDBmNzhmMjlmMTcyMDQ3NGY2YTZhOTRkODU2ZTgyYTUxNGEyNzk0MmQxZjZjMTIxMDU0OGExNzkzMjJiNDQmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.V5WrsxQpOaHPDtFIbRwfR62hq1WgtepfGcjVpodds1I)
> 
> It could even be put in a different section (new separator) cc @benjthayer for your ideas.
> 
> ### Acceptance Criteria
> * As an admin, right-clicking a member of a group chat in the member list shows the option to remove them from the group
> * Clicking that new option removes that user
> * Usual member removal stuff happens:
>   
>   * The user can no longer send messages to the chat or see new messages
>   * People see updated list of members

Nice idea - do you think we should make these controls dynamic so they change based on where the admin is accessing the profile menu from? 

I.e. if they are right-clicking a user in a group chat, we can include `Remove from group` but if they are right-clicking the user in the context of a community, we can include the options to `Kick user` and `Ban user`. I can see that being quite useful for quickfire moderation. 

I have a feeling these controls should be in their own section of the menu so they are more clearly delineated from the controls that are omnipresent across all contexts.

**Comment by jrainville** [comment]
> Nice idea - do you think we should make these controls dynamic so they change based on where the admin is accessing the profile menu from?

> I.e. if they are right-clicking a user in a group chat, we can include Remove from group but if they are right-clicking the user in the context of a community, we can include the options to Kick user and Ban user. I can see that being quite useful for quickfire moderation.

Good idea! I didn't think of the community context, but it makes a lot of sense!

> I have a feeling these controls should be in their own section of the menu so they are more clearly delineated from the controls that are omnipresent across all contexts.

I agree!

**Comment by benjthayer** [comment]
Hey @jrainville, flows for the community context menus containing Ban and Kick actions are now ready:  

https://www.figma.com/design/qHfFm7C9LwtXpfdbxssCK3/Kuba%E2%8E%9CDesktop---Communities?node-id=54395-99078&t=7RwDONOwdy7qOmoZ-4

cc @xAlisher

**Comment by jrainville** [comment]
@benjthayer looks good. @iurimatias I think you can do those changes in this issue as well, it's all the same component. The backend calls are different though, so if you prefer, you can create a new issue

**Comment by iurimatias** [comment]
Confirmed, I will take care of this issue once the ProfileContextMenu refactor is done
